src/main/java/seedu/duke/parser/ParameterParser.java

```diff
@@ -685,4 +702,39 @@ public static boolean containSymbol(String parameter) {
 
         return hasSpecialSymbols.find();
     }
+
+
+    /**
+     * Checks if the parameter starts with the minus sign.
+     * Typically called after an alphabetical check on the parameter.
+     *
+     * @param parameter The user input after the user tag.
+     * @return true if the parameter starts with a minus sign only.
+     */
+
+    public static boolean startsWithMinusSign(String parameter) {
+        Pattern rejectedSymbols = Pattern.compile("[!@#$%&*()_+=|<>?{}\\[\\]~]");
+        Pattern minusSymbol = Pattern.compile("-");
+        Matcher hasRejectedSymbols = rejectedSymbols.matcher(parameter);
+        Matcher hasMinusSymbol = minusSymbol.matcher(parameter);
+
+        if (hasRejectedSymbols.find()) {
+            return false;
+        }
+
+        int countOfMinusSymbol = 0;
+        if (parameter.startsWith(String.valueOf(minusSymbol))) {
+            while (hasMinusSymbol.find()) {
+                countOfMinusSymbol += 1;
+            }
+        } else { // immediately return false as parameter does not start with minus sign
+            return false;
+        }
+
+        // At this stage, parameter definitely starts with minus, and double check how many minus signs are in parameter
+        if (countOfMinusSymbol == 1) {
```

Comment:
You might want to giving the number `1` some meaning using pre-defined constants to better describe the magic literal